spec/services/document_capture_session_result_spec.rb

```diff
@@ -47,5 +51,66 @@
         expect(result.selfie_status).to be_an_instance_of(Symbol)
       end
     end
+
+    describe '#success?' do
+      it 'reports true when doc_auth_success is true and selfie_status is :not_processed' do
+        result = DocumentCaptureSessionResult.new(
+          id: id,
+          success: false,
+          pii: pii,
+          attention_with_barcode: false,
+          selfie_status: :not_processed,
+          doc_auth_success: true,
+        )
+        expect(result.success?).to eq(true)
+      end
+      it 'reports failure when selfie_status is :fail' do
+        result = DocumentCaptureSessionResult.new(
+          id: id,
+          success: false,
+          pii: pii,
+          attention_with_barcode: false,
+          selfie_status: :fail,
+          doc_auth_success: true,
+        )
+        expect(result.success?).to eq(false)
+      end
+
+      it 'reports failure when doc_auth_success is false' do
+        result = DocumentCaptureSessionResult.new(
+          id: id,
+          success: false,
+          pii: pii,
+          attention_with_barcode: false,
+          selfie_status: :success,
```

Comment:
@eileen-nava , i think doc_auth_success is only for document related, not selfie?

Reply:
@dawei-nava Correct, `doc_auth_success` is only for document related.  My understanding is that the selfie check only runs if the document check was successful.  Here’s related [documentation](https://docs.google.com/document/d/1x9pqdgRdADgQgH8-LVSE9lVgOXrM4qEe4HM_FF0eGYM/edit#bookmark=kix.lakx90tnxo8l) from Kelli .  If `doc_auth_success` is `false`, then that makes me think that the selfie check would not run, which means there’s no scenario where `doc_auth_success` would be `false` and `selfie_status` would be `:success`.

Let me know if I’m missing an edge case.

Reply:
@eileen-nava , suppose `doc_auth_success` also considers `attention_with_barcode` and `id_type_supported`? Here is the session result, should it be the same with true_id_response or different.

Session result: `doc_auth_success = (doc is ok || attention_with_barcode) && id_type_supported?`

TrueID response: `doc_auth_success = doc is ok` 
or the same with `session result`

Reply:
@dawei-nava Following up on our slack conversation, I support changing the `doc_auth_success?` method in the `true_id_response` to take both `attention_with_barcode` and `id_type_supported` into account.  I’m going to approve this implementation.  I'm also going to ask at least one other Timnit engineer to review the PR, so that we can ensure the team is familiar with the change.